src/main/java/testapp/core/User.java

```diff
@@ -1,30 +1,40 @@
 package testapp.core;
 
-public class User {
-    private int id;
+import com.google.appengine.api.datastore.Entity;
+
+import java.io.Serializable;
+
+public class User implements Serializable {
+    private long ucn;
     private String name;
```

Comment:
and why don't we use the id of the user? Why do we have to transmit the user's ucn over the network over and over again, and have it in every entity related to the user, what made you take that disition?